packages/web-components/fast-foundation/src/slider-label/slider-label.template.ts

```diff
@@ -13,17 +13,20 @@ export function sliderLabelTemplate(): ElementViewTemplate<FASTSliderLabel> {
             class="${x => x.sliderOrientation || Orientation.horizontal}
             ${x => (x.disabled ? "disabled" : "")}"
         >
-            <div ${ref("root")} part="root" class="root" style="${x => x.positionStyle}">
-                <div class="container">
-                    ${when(
-                        x => !x.hideMark,
-                        html`
-                            <div class="mark"></div>
-                        `
-                    )}
-                    <div class="label">
-                        <slot></slot>
-                    </div>
+            <div
+                ${ref("container")}
+                part="container"
+                class="container"
+                style="${x => x.positionStyle}"
+            >
+                ${when(
+                    x => !x.hideMark,
+                    html`
+                        <div class="mark"></div>
+                    `
+                )}
+                <div class="content">
+                    <slot></slot>
                 </div>
             </div>
```

Comment:
Curious if @marjonlynch recalls why/whether all this DOM is necessary...

Reply:
I don't remember too much about it but it seems like maybe the "content" div could be eliminated possibly, I suspect the "mark" div is necessary as the tick mark for the slider label, and the container is able to be styled externally with part and it's ref'd so probably code written against it, so it seems useable, maybe necessary? Probably could tighten it up a little without too much rework